lib/tasks/bundlers/generateFlexChangesBundle.js

```diff
@@ -69,8 +69,9 @@ module.exports = async function({workspace, taskUtil, options: {namespace}}) {
 		`${pathPrefix}/changes/*.{change,variant,ctrl_variant,ctrl_variant_change,ctrl_variant_management_change}`);
 	if (allResources.length > 0) {
 		const version = await readManifestMinUI5Version();
+		const versionArray = version.split(".");
 		let hasFlexBundleVersion = false;
-		if (parseFloat(version) >= 1.73) {
+		if (versionArray.length >= 2 && parseFloat(versionArray[0]) >= 1 && parseFloat(versionArray[1]) >= 73) {
```

Comment:
I'm just wondering if there is a version util class somewhere which does this

Reply:
I'd suggest to use semver, as mentioned in my comment above.